### DISCUSSION TITLE
What speed does Organic Maps assume you are biking at?

### DISCUSSION THREAD

**Discussion by unbeatable-101** [organicmaps/organicmaps#4952] [Q&A] [answered]
What speed does Organic Maps assume you are biking at when it gives you the ETA when you plan your route.

**Comment by biodranik** [comment, marked as answer]
Please check [routing_common/bicycle_model.cpp](https://github.com/organicmaps/organicmaps/blob/master/routing_common/bicycle_model.cpp) for details
